app/assets/stylesheets/mo/_utilities.scss

```
@@ -380,5 +421,4 @@
   .mt-sm-3 {
     margin-top: 1em !important;
   }
-}
-
+}
```

```suggestion
}

```
Insert newline at EOF